### ISSUE TITLE
Scylla stucks on stopping by command systemctl stop scylla-server.service

### ISSUE THREAD

**Issue by aleksbykov** [scylladb/scylla#7077]
*Installation details*
Scylla version (or git commit hash): 4.2.rc2-0.20200817.b052f3f5c
Cluster size: 5
OS (RHEL/CentOS/Ubuntu/AWS AMI): ami-0fa8d1a760f535d83(eu-north-1)

Node 4 was choose to run CorruptThenRebuild nemesis. First step of nemesis is stopping scylla.
But command sudo systemctl `stop scylla-server.service` was stucked and kill by timeout:
```
[centos@longevity-mv-si-4d-4-2-db-node-0301f62d-4 ~]$ ps aux | grep scylla
scylla      726  0.2  0.0 117724 13868 ?        Ssl  авг19   3:15 /usr/bin/node_exporter --collector.interrupts
scylla     5350  0.0  0.0 130468 26092 ?        Ssl  авг19   0:05 /usr/bin/scylla-manager-agent
centos    54343  0.0  0.0 112832  2544 pts/0    S+   09:24   0:00 grep --color=auto scylla
[centos@longevity-mv-si-4d-4-2-db-node-0301f62d-4 ~]$ sudo systemctl scylla-server status
Unknown operation 'scylla-server'.
[centos@longevity-mv-si-4d-4-2-db-node-0301f62d-4 ~]$ sudo systemctl status scylla-server
● scylla-server.service - Scylla Server
   Loaded: loaded (/usr/lib/systemd/system/scylla-server.service; enabled; vendor preset: disabled)
  Drop-In: /etc/systemd/system/scylla-server.service.d
           └─capabilities.conf, mounts.conf
   Active: failed (Result: signal) since Ср 2020-08-19 20:29:47 UTC; 12h ago
  Process: 25437 ExecStopPost=/opt/scylladb/scripts/scylla_stop (code=exited, status=0/SUCCESS)
  Process: 4996 ExecStart=/usr/bin/scylla $SCYLLA_ARGS $SEASTAR_IO $DEV_MODE $CPUSET $MEM_CONF (code=killed, signal=KILL)
 Main PID: 4996 (code=killed, signal=KILL)
   Status: "serving"

авг 19 20:13:42 longevity-mv-si-4d-4-2-db-node-0301f62d-4 scylla[4996]:  [shard 13] compaction_manager - compaction info: Compaction for sec_index/users was stopped due to: shutdown: stopping
авг 19 20:13:42 longevity-mv-si-4d-4-2-db-node-0301f62d-4 scylla[4996]:  [shard 13] compaction_manager - Stopped
авг 19 20:13:42 longevity-mv-si-4d-4-2-db-node-0301f62d-4 scylla[4996]:  [shard 9] compaction_manager - compaction info: Compaction for mview/users_by_email was stopped due to: shutdown: stopping
авг 19 20:13:42 longevity-mv-si-4d-4-2-db-node-0301f62d-4 scylla[4996]:  [shard 9] compaction_manager - compaction info: Compaction for sec_index/users was stopped due to: shutdown: stopping
авг 19 20:13:42 longevity-mv-si-4d-4-2-db-node-0301f62d-4 scylla[4996]:  [shard 9] compaction_manager - Stopped
авг 19 20:28:42 longevity-mv-si-4d-4-2-db-node-0301f62d-4 systemd[1]: scylla-server.service stop-sigterm timed out. Killing.
авг 19 20:29:46 longevity-mv-si-4d-4-2-db-node-0301f62d-4 systemd[1]: scylla-server.service: main process exited, code=killed, status=9/KILL
авг 19 20:29:47 longevity-mv-si-4d-4-2-db-node-0301f62d-4 systemd[1]: Stopped Scylla Server.
авг 19 20:29:47 longevity-mv-si-4d-4-2-db-node-0301f62d-4 systemd[1]: Unit scylla-server.service entered failed state.
авг 19 20:29:47 longevity-mv-si-4d-4-2-db-node-0301f62d-4 systemd[1]: scylla-server.service failed.
```

Latest messages from log:
```
2020-08-19T20:13:41+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 8] compaction - Compacted 2 sstables to [/var/lib/scylla/data/sec_index/users-7a73cf80e20711eabcb5000000000007/mc-8086-big-Data.db:level=0, ]. 876MB to 516MB (~58% of original) in 39219ms = 13MB/s. ~82816 total partitions merged to 48753.
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 8] compaction - Compacting [/var/lib/scylla/data/mview/users-69cd92b0e20711ea844f000000000000/mc-8380-big-Data.db:level=0, /var/lib/scylla/data/mview/users-69cd92b0e20711ea844f000000000000/mc-8366-big-Data.db:level=0, ]
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 8] compaction - Compacting [/var/lib/scylla/data/mview/users-69cd92b0e20711ea844f000000000000/mc-8324-big-Data.db:level=0, /var/lib/scylla/data/mview/users-69cd92b0e20711ea844f000000000000/mc-8352-big-Data.db:level=0, ]
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !NOTICE  | sudo:  centos : TTY=unknown ; PWD=/home/centos ; USER=root ; COMMAND=/bin/systemctl stop scylla-server.service
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 8] compaction - Compacted 4 sstables to [/var/lib/scylla/data/mview/users_by_address-722f9970e24e11ea9b25000000000003/mc-288-big-Data.db:level=0, ]. 25MB to 21MB (~84% of original) in 7293ms = 2MB/s. ~272896 total partitions merged to 231532.
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sudo: pam_unix(sudo:session): session opened for user root by (uid=0)
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd: Stopped Run Scylla Housekeeping daily mode.
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd: Stopped Run Scylla Housekeeping restart mode.
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd: Stopping Scylla JMX...
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !NOTICE  | systemd: scylla-jmx.service: main process exited, code=exited, status=143/n/a
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd: Stopped Scylla JMX.
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !NOTICE  | systemd: Unit scylla-jmx.service entered failed state.
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !WARNING | systemd: scylla-jmx.service failed.
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 0] compaction_manager - Asked to stop
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd: Stopping Scylla Server...
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 0] compaction_manager - Stopping 0 ongoing compactions
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 0] init - Signal received; shutting down
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 0] init - Shutting down redis service
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 0] init - Shutting down redis service was successful
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 0] init - Shutting down view builder
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 0] view - Stopping view builder
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 0] compaction_manager - Stopped
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 2] compaction_manager - Asked to stop
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 4] compaction_manager - Asked to stop
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 6] compaction_manager - Asked to stop
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 10] compaction_manager - Asked to stop
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 3] compaction_manager - Asked to stop
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 12] compaction_manager - Asked to stop
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 6] compaction_manager - Stopping 0 ongoing compactions
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 9] compaction_manager - Asked to stop
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 3] compaction_manager - Stopping 0 ongoing compactions
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 7] compaction_manager - Asked to stop
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 4] compaction_manager - Stopping 0 ongoing compactions
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 5] compaction_manager - Asked to stop
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 9] compaction_manager - Stopping 2 ongoing compactions
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 4] view - Stopping view builder
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 7] compaction_manager - Stopping 0 ongoing compactions
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 7] view - Stopping view builder
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 9] view - Stopping view builder
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 2] compaction_manager - Stopping 0 ongoing compactions
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 10] compaction_manager - Stopping 0 ongoing compactions
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 12] compaction_manager - Stopping 0 ongoing compactions
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 6] view - Stopping view builder
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 7] compaction_manager - Stopped
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 3] view - Stopping view builder
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 2] view - Stopping view builder
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 3] compaction_manager - Stopped
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 5] compaction_manager - Stopping 0 ongoing compactions
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 4] compaction_manager - Stopped
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 13] compaction_manager - Asked to stop
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 12] view - Stopping view builder
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 10] view - Stopping view builder
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 6] compaction_manager - Stopped
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 12] compaction_manager - Stopped
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 2] compaction_manager - Stopped
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 8] compaction_manager - Asked to stop
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 5] view - Stopping view builder
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 1] compaction_manager - Asked to stop
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 13] compaction_manager - Stopping 4 ongoing compactions
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 8] compaction_manager - Stopping 3 ongoing compactions
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 13] view - Stopping view builder
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 5] compaction_manager - Stopped
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 8] view - Stopping view builder
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 10] compaction_manager - Stopped
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 1] compaction_manager - Stopping 3 ongoing compactions
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 11] compaction_manager - Asked to stop
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 1] view - Stopping view builder
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 11] compaction_manager - Stopping 0 ongoing compactions
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 11] view - Stopping view builder
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 11] compaction_manager - Stopped
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 8] compaction_manager - compaction info: Compaction for mview/users was stopped due to: shutdown: stopping
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 8] compaction_manager - compaction info: Compaction for mview/users was stopped due to: shutdown: stopping
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 8] compaction_manager - compaction info: Compaction for sec_index/users_initials_ind_index was stopped due to: shutdown: stopping
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 8] compaction_manager - Stopped
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 13] compaction_manager - compaction info: Compaction for mview/users_by_address was stopped due to: shutdown: stopping
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 1] compaction_manager - compaction info: Compaction for sec_index/users_first_name_ind_index was stopped due to: shutdown: stopping
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 1] compaction_manager - compaction info: Compaction for sec_index/users was stopped due to: shutdown: stopping
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 1] compaction_manager - compaction info: Compaction for mview/users_by_password was stopped due to: shutdown: stopping
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 1] compaction_manager - Stopped
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 13] compaction_manager - compaction info: Compaction for sec_index/users was stopped due to: shutdown: stopping
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 13] compaction_manager - compaction info: Compaction for sec_index/users_last_access_ind_index was stopped due to: shutdown: stopping
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 13] compaction_manager - compaction info: Compaction for sec_index/users was stopped due to: shutdown: stopping
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 13] compaction_manager - Stopped
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 9] compaction_manager - compaction info: Compaction for mview/users_by_email was stopped due to: shutdown: stopping
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 9] compaction_manager - compaction info: Compaction for sec_index/users was stopped due to: shutdown: stopping
2020-08-19T20:13:42+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | scylla: [shard 9] compaction_manager - Stopped
2020-08-19T20:13:43+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16596]: Accepted publickey for centos from 10.0.2.169 port 44932 ssh2: RSA SHA256:g7OmDbWC8qLUZHxjMohQe3LsQHfcOJbG1ZHqodB6smc
2020-08-19T20:13:43+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd-logind: New session 6666 of user centos.
2020-08-19T20:13:43+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd: Started Session 6666 of user centos.
2020-08-19T20:13:43+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16596]: pam_unix(sshd:session): session opened for user centos by (uid=0)
2020-08-19T20:13:43+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16596]: pam_unix(sshd:session): session closed for user centos
2020-08-19T20:13:43+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd-logind: Removed session 6666.
2020-08-19T20:13:48+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16619]: Accepted publickey for centos from 10.0.2.169 port 44954 ssh2: RSA SHA256:g7OmDbWC8qLUZHxjMohQe3LsQHfcOJbG1ZHqodB6smc
2020-08-19T20:13:49+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd-logind: New session 6667 of user centos.
2020-08-19T20:13:49+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd: Started Session 6667 of user centos.
2020-08-19T20:13:49+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16619]: pam_unix(sshd:session): session opened for user centos by (uid=0)
2020-08-19T20:13:49+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16619]: pam_unix(sshd:session): session closed for user centos
2020-08-19T20:13:49+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd-logind: Removed session 6667.
2020-08-19T20:13:51+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !NOTICE  | sudo:  centos : TTY=unknown ; PWD=/home/centos ; USER=root ; COMMAND=/bin/coredumpctl --no-pager --no-legend
2020-08-19T20:13:51+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sudo: pam_unix(sudo:session): session opened for user root by (uid=0)
2020-08-19T20:13:51+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sudo: pam_unix(sudo:session): session closed for user root
2020-08-19T20:13:54+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16663]: Accepted publickey for centos from 10.0.2.169 port 44978 ssh2: RSA SHA256:g7OmDbWC8qLUZHxjMohQe3LsQHfcOJbG1ZHqodB6smc
2020-08-19T20:13:54+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd-logind: New session 6668 of user centos.
2020-08-19T20:13:54+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd: Started Session 6668 of user centos.
2020-08-19T20:13:54+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16663]: pam_unix(sshd:session): session opened for user centos by (uid=0)
2020-08-19T20:13:54+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16663]: pam_unix(sshd:session): session closed for user centos
2020-08-19T20:13:54+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd-logind: Removed session 6668.
2020-08-19T20:13:59+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16686]: Accepted publickey for centos from 10.0.2.169 port 44996 ssh2: RSA SHA256:g7OmDbWC8qLUZHxjMohQe3LsQHfcOJbG1ZHqodB6smc
2020-08-19T20:14:00+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd-logind: New session 6669 of user centos.
2020-08-19T20:14:00+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd: Started Session 6669 of user centos.
2020-08-19T20:14:00+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16686]: pam_unix(sshd:session): session opened for user centos by (uid=0)
2020-08-19T20:14:00+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16686]: pam_unix(sshd:session): session closed for user centos
2020-08-19T20:14:00+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd-logind: Removed session 6669.
2020-08-19T20:14:05+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16709]: Accepted publickey for centos from 10.0.2.169 port 45016 ssh2: RSA SHA256:g7OmDbWC8qLUZHxjMohQe3LsQHfcOJbG1ZHqodB6smc
2020-08-19T20:14:05+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd-logind: New session 6670 of user centos.
2020-08-19T20:14:05+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd: Started Session 6670 of user centos.
2020-08-19T20:14:05+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16709]: pam_unix(sshd:session): session opened for user centos by (uid=0)
2020-08-19T20:14:06+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16709]: pam_unix(sshd:session): session closed for user centos
2020-08-19T20:14:06+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd-logind: Removed session 6670.
2020-08-19T20:14:06+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !NOTICE  | sudo:  centos : TTY=unknown ; PWD=/home/centos ; USER=root ; COMMAND=/bin/coredumpctl --no-pager --no-legend
2020-08-19T20:14:06+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sudo: pam_unix(sudo:session): session opened for user root by (uid=0)
2020-08-19T20:14:06+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sudo: pam_unix(sudo:session): session closed for user root
2020-08-19T20:14:11+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16744]: Accepted publickey for centos from 10.0.2.169 port 45034 ssh2: RSA SHA256:g7OmDbWC8qLUZHxjMohQe3LsQHfcOJbG1ZHqodB6smc
2020-08-19T20:14:11+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd-logind: New session 6671 of user centos.
2020-08-19T20:14:11+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd: Started Session 6671 of user centos.
2020-08-19T20:14:11+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16744]: pam_unix(sshd:session): session opened for user centos by (uid=0)
2020-08-19T20:14:11+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16744]: pam_unix(sshd:session): session closed for user centos
2020-08-19T20:14:11+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd-logind: Removed session 6671.
2020-08-19T20:14:16+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16767]: Accepted publickey for centos from 10.0.2.169 port 45058 ssh2: RSA SHA256:g7OmDbWC8qLUZHxjMohQe3LsQHfcOJbG1ZHqodB6smc
2020-08-19T20:14:17+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd-logind: New session 6672 of user centos.
2020-08-19T20:14:17+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd: Started Session 6672 of user centos.
2020-08-19T20:14:17+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16767]: pam_unix(sshd:session): session opened for user centos by (uid=0)
2020-08-19T20:14:17+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16767]: pam_unix(sshd:session): session closed for user centos
2020-08-19T20:14:17+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | systemd-logind: Removed session 6672.
2020-08-19T20:14:21+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | dhclient[1150]: PRC: Renewing lease on eth0.
2020-08-19T20:14:21+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | dhclient[1150]: XMT: Renew on eth0, interval 9150ms.
2020-08-19T20:14:21+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | dhclient[1150]: RCV: Reply message on eth0 from fe80::4be:8fff:fef6:4fe0.
2020-08-19T20:14:21+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !NOTICE  | sudo:  centos : TTY=unknown ; PWD=/home/centos ; USER=root ; COMMAND=/bin/coredumpctl --no-pager --no-legend
2020-08-19T20:14:21+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sudo: pam_unix(sudo:session): session opened for user root by (uid=0)
2020-08-19T20:14:21+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sudo: pam_unix(sudo:session): session closed for user root
2020-08-19T20:14:22+00:00  longevity-mv-si-4d-4-2-db-node-0301f62d-4 !INFO    | sshd[16818]: Accepted publickey for centos from 10.0.2.169 port 45080 ssh2: RSA SHA256:g7OmDbWC8qLUZHxjMohQe3LsQHfcOJbG1ZHqodB6smc
```

Node is available: centos@13.53.127.51
[system.log.tar.gz](https://github.com/scylladb/scylla/files/5102431/system.log.tar.gz)

**Comment by slivne** [comment]
@bhalevy can you have someone look at this - thanks - the node is avail

**Comment by xemul** [comment]
@aleksbykov , you can take the node, thanks.

**Comment by xemul** [comment]
Last shard 0 stopping messages:

````
  [shard 0] compaction_manager - Stopping 0 ongoing compactions
  [shard 0] init - Signal received; shutting down
  [shard 0] init - Shutting down redis service
  [shard 0] init - Shutting down redis service was successful
  [shard 0] init - Shutting down view builder
  [shard 0] view - Stopping view builder
  [shard 0] compaction_manager - Stopped
````

**Comment by xemul** [comment]
All shards went through
```
Stopping view builder
```
message, but not any further.

**Comment by avikivity** [comment]
@xemul how far back does this need to be backported?

**Comment by xemul** [comment]
It was intruduced by 996b86b8, the next tag after this commit is scylla-3.0.0

**Comment by avikivity** [comment]
This is a fairly drastic change. I think it should be backported, but only after a lot more soak time. Keeping the Backport Candidate label.

**Comment by slivne** [comment]
Backport to 4.2 deferred - @roydahan / @aleksbykov if you hit this again - please update (on 4.2.rc4)

**Comment by avikivity** [comment]
Backported to 4.2 due to #7332.

**Comment by avikivity** [comment]
No further backports are neeed.
